packages/x-date-pickers/src/internals/hooks/useField/useField.ts

```diff
@@ -403,9 +403,13 @@ export const useField = <
     return 'tel';
   }, [selectedSectionIndexes, state.sections]);
 
+  const inputHasFocus = inputRef.current && inputRef.current === document.activeElement;
+
   return {
     ...otherForwardedProps,
-    value: valueStr,
+    placeholder: !state.value ? valueStr : undefined,
```

Comment:
Is there a downside in always passing the `placholder` ? When there is a value, it will be ignored

Reply:
Good point. Will test it, but should work fine. 👌

Reply:
Ah, yes, I remember now. I did not want to put pointless `placeholder` prop updating, because it would mimic the value, instead of actually being a `placeholder`, when the value is present. 🙈 
![Screenshot 2023-02-28 at 13 17 33](https://user-images.githubusercontent.com/4941090/221838738-74055a7f-fb22-4982-b37f-d538ddf9cab5.png)

Reply:
OK it make sense
We could generate a static placeholder even when the value is not empty but that's probably not worth the effort

Reply:
I was actually thinking about the same thing.
It would make more sense to always provide a placeholder, without needlessly making additional DOM modifications. 🤔

Reply:
[Added it](https://github.com/mui/mui-x/pull/8069/commits/06008c80552f9379c2d05cd632ba97dc40aa8443). 😉